### PR TITLE
#160577601 Integrates TravisCI with readme badge

### PR DESCRIPTION
**Pull request by C3real-kill3r**
## What does this PR do?
* It integrates TravisCI with readme badge

## Why is this important?
* This is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product

## Relevant PT story.
[160577601](https://www.pivotaltracker.com/story/show/160577601)

### CODE REVIEW

**Comment by esirK** [comment]
Looks good but you need to include the badge in the readme.

**Comment by C3real-kill3r** [comment]
I have insufficient permission to integrate it...

**Comment by esirK** [comment]
> I have insufficient permission to integrate it...

I have given you admin permission.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/andela/ah-code-blooded.svg?branch=master)](https://travis-ci.org/andela/ah-code-blooded)
```
Comment (esirK) [inline README.md]: let the branch be develop

**Comment by esirK** [comment]
Kindly resolve the merge conflict.

**Comment by C3real-kill3r** [comment]
> Kindly resolve the merge conflict.

Is it okay now?

**Comment by esirK** [comment]
> > Kindly resolve the merge conflict.
> 
> Is it okay now?

I don't think so.

**Comment by C3real-kill3r** [comment]
> > > Kindly resolve the merge conflict.
> > 
> > 
> > Is it okay now?
> 
> I don't think so.

It's now okay

**Comment by esirK** [comment]
> > > > Kindly resolve the merge conflict.
> > > 
> > > 
> > > Is it okay now?
> > 
> > 
> > I don't think so.
> 
> It's now okay

Fix the failing test now.

> > > > Kindly resolve the merge conflict.
> > > 
> > > 
> > > Is it okay now?
> > 
> > 
> > I don't think so.
> 
> It's now okay

Sure. But kindly fix the failing builds.

**Comment by C3real-kill3r** [comment]
> > > > > Kindly resolve the merge conflict.
> > > > 
> > > > 
> > > > Is it okay now?
> > > 
> > > 
> > > I don't think so.
> > 
> > 
> > It's now okay
> 
> Fix the failing test now.
> 
> > > > > Kindly resolve the merge conflict.
> > > > 
> > > > 
> > > > Is it okay now?
> > > 
> > > 
> > > I don't think so.
> > 
> > 
> > It's now okay
> 
> Sure. But kindly fix the failing builds.

On it.

**Comment by gitaumoses4** [comment]
Hello @C3real-kill3r 
Good work on the Travis integration. I would advise that you ensure your commits indicate the actual task you worked on. Having a similar commit message for many commits may be a bit misleading.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+service:
+  - postgresql
+
+install:
+  - pip3 install -r requirements.txt
+  - pip3 install coverage
+  - pip3 install coveralls
+
+before_script:
+  - export DB_NAME="ah_code_blooded"
```
Comment (gitaumoses4) [inline .travis.yml:15]: For Django, you will not need to set up any database settings on Travis

**Review by gitaumoses4 — APPROVED** [review]
The integration seems good to me. 
Work on the current feedback and all will be well. 👍

**Comment by bevkololi** [comment]
Seems good to me too. Well done @C3real-kill3r ! We need to work on some of these errors though, @gitaumoses4

**Comment by esirK** [comment]
LGTM. Good job. For the failing tests, @gitaumoses4, @bevkololi  check them out.

**Pull request merged**